pillar.example

```diff
@@ -33,6 +33,11 @@ gitlab:
       # Run at a specific time, defaults to 02:00 every night
       on_calendar: 02:00
 
+  gitaly:
+    signing_key:
```

Comment:
Mind to add two examples here? One with `ssh-ed25519 AAA...` and one with a dummy GPG key?

Reply:
Added!